### ISSUE TITLE
Currencies

### ISSUE THREAD

**Issue by chKaze** [chKaze/Xianxia-Incremental-Game#4]
On behalf of reddit user /u/cocoblarog

> I have been waiting for this for god knows how long. DO ITTTTT.
Things that would be nice : Laws (like random drops or wtv into a law, with X random drop, you can buy lets say a fireball skill that triggers with a %, then with more law drop, you can upgrade it)
Pills : That one is easy, just have it give some cultivation exp I guess
Monster cores : could use that as currency, no need to make it complicated, have like monsters from lvl 1-10 drop cores1, then 10-20 cores2, etc, and then a core3 is worth 3 core2 which are worth 3 core1 each or something. Sorry too hyped to think about balance.
Got more ideas but not sober enough atm, I had a whole bunch of ideas to make a pbbg based on xianxia, but sadly I am too busy with school to learning coding on top of it.
TL;DR : GO FOR IT, make it happen!

**Comment by chKaze** [comment]
Going to implement the following for v0.4.0

## Mortal Currency:
**Coins:**
100:1 Ratio
Copper -> Silver -> Gold -> Platin -> Orichalcum

Used for buying lowgrade alchemy items, slaves, buildings etc.
## Cultivator Currency
**Spirit Stones:**
100:1 Ratio
Grade: Tainted -> Low -> Mid ->High -> Ultra-High

Can be used as a cultivation resource or currency for mid to high grade items.

**Monster Cores:**
100:1 Ratio
Tier: 1 -> 2 -> 3 -> 4 -> 5 ->6 -> 7 -> 8 -> 9

Can be used as a cultivation resource or item in alchemy.
